tests/connect_redirect/connect_redirect_tests.cpp

```diff
@@ -416,6 +416,49 @@ update_policy_map_and_test_connection(
         destination, proxy, destination_port, proxy_port, _globals.connection_type, dual_stack, add_policy);
 }
 
+void
+authorize_connection_test(
```

Comment:
This looks similar to update_policy_map_and_test_connection - do we really need two separate functions?

Reply:
please take a look at this comment.